src/client/styles/common.less

```diff
@@ -37,7 +37,7 @@ a:focus {
 }
 
 .ant-layout {
-  overflow: hidden;
+  overflow: visible;
```

Comment:
Isn't `visible` the default?

Reply:
Nah it's never specified, for `.ant-layout` but default for all html elements is `visible`. Thought it would be good to explicitly set it.